explorer/package.json

```diff
@@ -7,41 +7,34 @@
   "license": "MIT",
   "keywords": [],
   "scripts": {
-    "predev": "yarn autosetup",
+    "addclnode": "ts-node src/bin/clnodes.ts add",
+    "admin:seed": "ts-node src/bin/admin.ts seed",
+    "depcheck": "echo \"@chainlink/explorer\" && depcheck --ignore-dirs=client || exit 0",
+    "predev": "yarn automigrate",
     "dev": "concurrently \"cd client && yarn start\" \"ts-node-dev --respawn --transpileOnly ./src/index.ts\"",
     "dev:client": "yarn workspace @chainlink/explorer-client run start",
     "dev:server": "ts-node-dev --respawn --transpileOnly ./src/index.ts",
     "prebuild": "yarn install",
     "build": "yarn workspace @chainlink/explorer-client run build",
     "prod": "yarn migration:run && ts-node ./src/index.ts",
-    "pretest": "yarn autosetup",
+    "pretest": "NODE_ENV=test yarn automigrate",
```

Comment:
might need cross-env on here if we plan to support windows developers

Reply:
Added to `automigrate`